docs/docker/README.md

```diff
@@ -0,0 +1,22 @@
+# Running Graphile Migrate from a container
+
+When working in a team it can be useful to package `graphile-migrate` and the
+postgres tooling into a docker file so that everyone has easy access to the
+same versions. This is also helpful if you're using Migrate as part of a larger
+non-nodejs based project.
```

Comment:
```suggestion
When working in a team it can be useful to package `graphile-migrate` and the
Postgres tooling into a Docker file so that everyone has easy access to the
same versions. This is also helpful if you're using Migrate as part of a larger
non-Node based project.
```